lib/pharos/phase_manager.rb

```diff
@@ -64,7 +64,13 @@ def prepare_phase(phase_class, host, ssh: false, **options)
     def apply(phase_class, hosts, parallel: false, **options)
       phases = hosts.map { |host| prepare_phase(phase_class, host, **options) }
 
-      run(phases, parallel: parallel, &:call)
+      run(phases, parallel: parallel) do |phase|
```

Comment:
(I still don't see why the runner doesn't do the calling, it's already doing all sorts of magic with `prepare_phase` method for the `phases`, so it obviously expects the `phases` to be phases, which can be `.call`ed)

Reply:
`prepare_phase` is used from `apply`, which is also the same method that knows about `call`?